language/src/main.rs

```diff
@@ -773,6 +832,21 @@ fn main() -> Result<(), usize> {
         None => None,
     };
 
+    let init_file = match args.iter().position(|a| a == "--init") {
```

Comment:
If the arguments are only `init` and `update`, it's not entirely clear what they refer to.
Also the other args use only one `-`.

Maybe something like `-vcs-init` could work?

Reply:
usually you use `-` for one letter arguments, while you use `--` for longer arguments. I guess we should update dir to be `--` in that case.